### ISSUE TITLE
Autocomplete css styling is missing

### ISSUE THREAD

**Issue by mrisedev** [unfoldadmin/django-unfold#216]
Please add css for autocompleted fields background.
![Screenshot_1](https://github.com/unfoldadmin/django-unfold/assets/129842335/f1496de8-c983-4b51-90df-579caea968a2)

**Comment by lukasvinclav** [comment]
@mrisedev would you provide minimal example how to replicate the issue?

**Comment by mrisedev** [comment]
@lukasvinclav
I don't think I can, cuz it's a browser(chrome) feature.
The simplest way to reproduce would be to go to chrome aftofill settings and create an address with some email.
https://nordpass.com/blog/edit-remove-autofill-chrome/
Then create some django modeladmin form with 'email' field.
Also add some 'name' field mb. If attempt with "email" dosent work, just create a few objs with different names and email and chrome will start to show you autofill suggestions.
![image](https://github.com/unfoldadmin/django-unfold/assets/129842335/718458aa-523e-40a8-ade3-77849c86ed5b)

**Comment by mrisedev** [comment]
![image](https://github.com/unfoldadmin/django-unfold/assets/129842335/5521d007-4212-46d0-8b3f-523d288d75ec)
screenshot from a demo website

**Comment by mrisedev** [comment]
input:-webkit-autofill, input:-webkit-autofill:hover, input:-webkit-autofill:focus, input:-webkit-autofill:active {
    -webkit-background-clip: text;
    -webkit-text-fill-color: #9e9e9e;
    transition: background-color 5000s ease-in-out 0s;
}

Not ideal, but works.

**Comment by lukasvinclav** [comment]
I was able to replicate this issue and after some investigation I found out there is no elegant way how to solve this issue. I was not able to find a way how to override text or background color without using box-shadow or text-fill. At the top of that, I noticed font-family is different for autofilled elements as well. There is an article describing probably this issue:

https://www.stefanjudis.com/today-i-learned/you-cant-override-important-user-agent-css-declarations/

@mrisedev @Rey092 Right now, I would like to close this issue and come back to it later when clean solution will be available.
